### DISCUSSION TITLE
[Select] Is it possible to set the `radius` attribute on `SelectContent` component?

### DISCUSSION THREAD

**Discussion by lamualfa** [radix-ui/themes#114] [Q&A] [answered]
### Screenshot

![image](https://github.com/radix-ui/themes/assets/39755201/84e65f12-c2d7-466d-964a-74036b7c2ae2)

### Code

```tsx
export function SelectField() {
  return (
    <SelectRoot>
      <SelectTrigger placeholder='Select one' radius='medium' />
      <SelectContent>
        <SelectItem value='1'>Option 1</SelectItem>
        <SelectItem value='2'>Option 2</SelectItem>
        <SelectItem value='3'>Option 3</SelectItem>
        <SelectItem value='4'>Option 4</SelectItem>
      </SelectContent>
    </SelectRoot>
  )
}
```

**Comment by vladmoroz** [comment, marked as answer]
Hey, for panels like Select and Dropdown Menu Content, Dialogs, Card, Table, the radius is controlled only via the Theme radius prop, as most of the time you want all panels to feel similar. You can override the radius via own prop only for form components like Button or Text Field.

**Reply by lamualfa** [reply]
Oo, I see. Thank you very much.
